src/entrypoint.rs

```diff
@@ -11,13 +10,7 @@ fn process_instruction(
     accounts: &[AccountInfo],
     instruction_data: &[u8],
 ) -> ProgramResult {
-    msg!(
```

Comment:
In order to test your BPF program, it needs to call into your instruction processor